db/migrate/20250827203528_add_event_groups.rb

```diff
@@ -0,0 +1,19 @@
+class AddEventGroups < ActiveRecord::Migration[7.2]
+  def change
+    create_table(:event_groups) do |t|
+      t.citext(:name, null: false)
+      t.references(:user, null: false, foreign_key: true, index: false)
+      t.timestamps
+
+      t.index([:user_id, :name], unique: true)
```

Comment:
I have a feeling we want groups to be shared across all admins. There's a very likely case where Mel/Ops will create a group for Scrapyard, and then it's later shared with Zach, etc., who can easily grab reporting for that group.

I'd suggest we do a similar structure as Blazer queries. Any admin can create one. It says who created it. Anyone can edit. Anyone can view the list of queries.

We can enforce unique names across all event groups (not just scoped to user)